apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/GetSchemaTableTests.cs

```diff
@@ -45,11 +44,15 @@ public async Task GetSchemaTable_WithFlagEnabled_ReturnsNull()
         {
             using (var connection = new SpannerConnection($"{_fixture.ConnectionString};EnableGetSchemaTable=true"))
             {
-                var command = connection.CreateSelectCommand($"SELECT Int64Value, BytesArrayValue FROM {_fixture.TableName}");
+                var sql = _fixture.RunningOnEmulator
```

Comment:
I think it would be cleaner to have a separate to for JSON schema values. Most of what's tested is here doesn't really correspond with "GetSchemaTable with the flag enabled returns null".

Reply:
I think that this test case more in general could need some work:
1. The `_ReturnsNull` suffix indicates that the method should return `null` with the flag enabled, but it actually does the opposite: Enabling the flag will ensure that the method does __not__ return null.
2. The current method only checks the return values of `Int64` and `BytesArray` columns. I think it would be valuable to cover all the data types in this test case. It already uses a table that contains columns with all supported data types, so it would only be a matter of adding assertions for the other columns as well. (This is the reason I chose to add the assertions for JSON here)

WDYT?

Reply:
Yes, that shouldn't be a problem, it will probably take a little longer before this PR can be merged.

(I'm also happy to take it on, if that would be easier for you)

Reply:
If you've got the time, that would be great, thanks. Feel free to change your mind and push it back to me :)